src/main/java/eu/donals/Player.java

```diff
@@ -0,0 +1,88 @@
+package eu.donals;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Player {
+    private String name;
+    private List<String> inventory;
+    private UserInterface ui;
+
+    public boolean hasItem(String item) { return inventory.contains(item); }
+
+    public String getPlayerName() { return name; }
```

Comment:
I would rename getPlayerName to getName, because the resulting function calls like player.getName() are less redundant.